### PR TITLE
Fix image build and use imagename

### PR DESCRIPTION
**Pull request by cdarwin**

### CODE REVIEW

File: launch.sh [hunk 1]
```diff
@@ -37,9 +38,9 @@ test -n "${API_KEY}" && \
     api_key=${API_KEY}
 
 test -n "${AMPLIFY_HOSTNAME}" && \
```
Comment (dedm) [inline launch.sh]: let's use AMPLIFY_IMAGENAME here

File: Dockerfile [hunk 2]
```diff
@@ -1,53 +1,44 @@
 FROM nginx:latest
+MAINTAINER NGINX Aplify Engineering
```
Comment (dedm) [inline Dockerfile]: typo

File: launch.sh (hunk 1 above)
Comment (cdarwin) [inline launch.sh]: I'm ok with this. Especially since this is a new behaviour.

File: Dockerfile (hunk 2 above)
Comment (cdarwin) [inline Dockerfile]: fixed. thanks!

**Comment by ptreyes** [comment]
Closing this one. Changes incorporated as part of
https://github.com/ptreyes/docker-nginx-amplify/tree/improve_build

**Pull request closed without merging**